pytorch_lightning/core/lightning.py

```diff
@@ -855,11 +855,15 @@ def init_ddp_connection(self):
         try:
             root_node = os.environ['SLURM_NODELIST'].split(' ')[0]
         except Exception:
-            root_node = '127.0.0.2'
+            root_node = '127.0.0.1'
```

Comment:
why the IP change?

Reply:
I wish I knew. 127.0.0.1 works, 127.0.0.2 doesn't.